contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,354 @@
+> * What sort of company do you want to run? How would it look and feel day to day?
+> * What roles would we each have? Will you feel happy about this in 5 years?
+> * Would you sell for $1M / $10M / $100M / $1Bn?
+> * What kind of work do you want to do / what don't you want to do?
+
+### A simple, well-designed website
+
+Website design is probably more important than you think.
+
+When it comes to the website, most people will grab a template to get something thrown together quickly.
+
+Cory, PostHog's lead designer, and self-proclaimed webmaster, put this really nicely "we didn't start by thinking we could build the slickest website, so we built one that felt different instead".
+
+When it comes to the product, you want to ship a competent-feeling product, but in most cases, it doesn't need to be best-in-class. I'd argue that your website's design is _more_ important when you're in a phase of trying to get anyone to sign up, since it's probably where you're losing 90% or more of potential signups.
+
+The good news is that design starts with copy. If you get simple, clear language onto your website that explains what you _do_, and meets the other use cases your visitors may have, then you're ahead of 75% of other websites.
+
+Don't worry about pixel perfection. I once met (Malthe Sigurdsson)[https://twitter.com/malthe], who ran design at Stripe. I went into the conversation expecting to hear that they spent months and months in Figma before shipping anything. Instead, he told me that great designers are fast and iterative. They don't get things perfect first time around.
+
+### A "build fast" mentality
+
+As with most early-stage startup stuff, you're more likely to be wrong than right (the vast majority of startups fail, after all), so optimize for this scenario.
+
+You are trying to figure out if anyone cares, so the speed you can ship your first feature is the most important factor. Scalability is irrelevant. Use a software stack you've used before, as long as it's reasonably popular. We used Django and React in the early days.
+
+Ship features embarassingly early. The kind of user that'll pick up a random piece of software that no one has ever used before is an early adopter. 
+
+Unless your software's main priority is being better designed, it's preferable to ship clunky/ugly/slightly broken functionality and quickly improve it, than to waste months on something no one cares about.
+
+### An open and responsive attitude 
+
+Aside from closing the loop, don't put anyone or anything between the person writing the code and your users. This will slow you down and will cause confusion. Moreover, it's likely a better experience for a user to spend time with the person that actually built the product. It is more effective in the long run to productize answers to users' problems, than it is to spent lots of time triaging issues... which wind up with you or your engineers anyway.
```

Comment:
```suggestion
Aside from closing the loop, don't put anyone or anything between the person(s) writing the code and your users. This will slow you down and will cause confusion. Moreover, it's likely a better experience for a user to spend time with the person that actually built the product. It is more effective in the long run to productize answers to users' problems, than it is to spent lots of time triaging issues.
```